deepmd/pt/loss/population.py

```
@@ -0,0 +1,208 @@
+    DataRequirementItem,
+)
+
+from functools import partial
+
+log = logging.getLogger(__name__)
+
+class PopulationLoss(TaskLoss):
+    def __init__(
+        self,
+        loss_func: str = "smooth_mae",
+        metric: list = ["mae"],
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)